web/Views/Landlord/Properties.cshtml

```diff
@@ -7,23 +7,35 @@
 <h2>My properties</h2>
 
 <table class="table">
-  <thead>
+    <thead>
     <tr>
-      <th scope="col">#</th>
-      <th scope="col">Address</th>
+        <th scope="col">#</th>
+        <th scope="col">Address</th>
+        <th scope="col">Type</th>
+        <th scope="col"># of Bedrooms</th>
+        <th scope="col">Created</th>
+        <th scope="col">Rent (pcm)</th>
+        <th scope="col">Description</th>
     </tr>
-  </thead>
-  <tbody>
-  @{
-    int i = 1;
-  }
-  @foreach (var prop in Model.Properties)
-  {
-    <tr>
-      <th scope="row">@i</th>
-      <td>@prop.Address</td>
-    </tr>
-    i++;
-  }
-  </tbody>
-</table>
\ No newline at end of file
+    </thead>
+    <tbody>
+    @{
+        var i = 1;
+    }
+    @foreach (var prop in Model.Properties)
+    {
+        <tr>
+            <th scope="row">@i</th>
+            <td>@prop.Address!.AddressLine1, @prop.Address.AddressLine2, @prop.Address.AddressLine3, @prop.Address.TownOrCity</td>
```

Comment:
We can add some null checks so that we don't display "Line 1, , , City" when address lines 2 and 3 are empty. You might update how the values are stored so that they save as null as opposed to empty string when the input textbox is empty.

Do let me know if this is something better addressed in BNH-34, then we can get this merged first 🙂

Reply:
Yeah I think the nullability of most fields changes in BNH-34, so it might be easier to do it there! 🙂